src/libraries/System.ComponentModel.Composition.Registration/tests/System/ComponentModel/Composition/Registration/RegistrationBuilderAttributedOverrideUnitTests.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace System.ComponentModel.Composition.Registration.Tests
 {
-    [SkipOnCoreClr("Test failures on stress tests")]
+    [SkipOnCoreClr("Test failures on stress tests", RuntimeTestModes.CheckedRuntime)]
```

Comment:
Is it just checked builds, or any of the stress / non-release configurations?

Reply:
This would apply to ALL Checked builds, no matter if it is stress or not. 

However, I just realized that the current design is not great as if we added a `| RuntimeTestModes.GCStress` it would mean, ALL Checked or GCStress (Checked or Release), as I would think what we want is that if I say `RuntimeTestModes.Checked` it would mean, ALL Checked, but if I add an stress mode flag, then it would mean, ONLY skip when running `GCStress` when in `Checked`. What is your expectation/thought for that?

Reply:
@stephentoub thoughts on that?

I'll include this change in the arcade update to unblock and can update again based on your thoughts. I think updating it to the latter is the ideal.

Reply:
Actually, I gave it a thought and put up a PR to update it to a better way of doing it: https://github.com/dotnet/arcade/pull/4764